src/main/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2.java

```diff
@@ -61,13 +63,12 @@ public RedirectView login(final @RequestParam Optional<String> redirectUrl,
                     new Cookie(REDIRECT_URL_COOKIE, redirectUrl.orElse(configProperties.getDefaultRedirectUrl())),
                     Boolean.TRUE,
                     Boolean.TRUE,
-                    null
-            );
+                    null);
 
             response.addCookie(redirectUrlCookie);
 
             // TODO check if this is the correct URL
-            return new RedirectView(oneLoginBaseUrl);
+            return new RedirectView(NOTICE_PAGE_VIEW);
```

Comment:
Does this mean that returning users will always see this screen? I would have thought they would only see this the first time.